README.md

```diff
@@ -1,4 +1,4 @@
-# Q-CTRL Open Controls
+# Open Controls
 
 Q-CTRL Open Controls is an open-source Python package that makes it easy to
```

Comment:
```suggestion
Open Controls is an open-source Python package that makes it easy to
```
Can we also remove Q-CTRL from the rest part of the README?

Reply:
If I remember correctly the only rule is that the title of the `README.md` has to match the Package name. I don't have strong opinions about changing the rest of the text, but I'm happy to do it if you prefer.

Reply:
Not really, I *thought* there is a rule about updating them in all places. If not, I am fine with what you have here.